app/javascript/copy_to_clipboard.js

```diff
@@ -0,0 +1,17 @@
+window.addEventListener('load', function() {
```

Comment:
@marlo-longley this doesn't seem to work if there's more than one token on the page. I can copy one of them, but if I then try to copy the other one it doesn't work.

Reply:
Ohhh that makes total sense. I obviously should have but only had 1 token on my local and didn't account for that. Thank you! @corylown